hawc/apps/lit/api.py

```diff
@@ -385,3 +386,20 @@ def tag(self, request, pk):
             ref.update_tags(request.user, tag_pks)
             response["status"] = "success"
         return Response(response)
+
+    @action(detail=True, methods=("post",))
+    def resolve_conflict(self, request, pk):
+        template = "lit/_reference_tag_conflict.html"
+        errors = None
+        ref = self.get_object()
+        assessment = ref.assessment
+        if assessment.user_can_edit_object(self.request.user):
+            try:
+                user_tag_id = request.POST.get("user_tag_id")
+                ref.resolve_user_tag_conflicts(user_tag_id)
+                template = "lit/_conflict_resolved.html"
+            except Exception:
```

Comment:
global catch all exceptions are generally a pattern we should avoid if possible

Reply:
I simplified this to just throw 403 or 404 instead of nicely showing exceptions - these issues wouldn't be shown by users in general